app/javascript/components/payment-error-button.vue

```diff
@@ -2,18 +2,25 @@
   <div class="payment-error-button">
     <modal name="apology-modal">
       <div class="invoice__apology-modal-text">
-        <h1>Error en el pago</h1>
-        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+        <div v-if="showApologyText">
+          <h1>{{ apologyTextTitle }}</h1>
+            {{ apologyText }}
+            <cartProducts v-if="bla"/>
+        </div>
       </div>
       <div
         class="invoice__apology-modal-debtn"
       >
         <button
           type="button"
           class="btn"
+          v-if="showDebtButton"
+          @click="debtorNameInput"
         >
           Fiar
         </button>
+        <input v-model="message" placeholder="Tu Nombre" v-if="showNameTextBox">
+        <button type="button" class="btn" v-if="showNameTextBox" @click="confirmDebtButton">Confirmar</button>
```

Comment:
Attribute "class" should be on a new line                              vue/max-attributes-per-line<br>Attribute "if" should be on a new line                                 vue/max-attributes-per-line<br>Attribute "on" should be on a new line                                 vue/max-attributes-per-line